{"name": "Verify error message for login with incorrect credentials -- @1.1 ", "status": "passed", "steps": [{"name": "Given the user is on the login page", "status": "passed", "start": 1751361197409, "stop": 1751361198135}, {"name": "When the user enter the standard_user and invalid_password", "status": "passed", "start": 1751361198135, "stop": 1751361198243}, {"name": "Then the Epic sadface: Username and password do not match any user in this service is displayed", "status": "passed", "start": 1751361198243, "stop": 1751361198252}], "parameters": [{"name": "username", "value": "standard_user"}, {"name": "password", "value": "invalid_password"}, {"name": "error message", "value": "Epic sadface: Username and password do not match any user in this service"}], "start": 1751361197289, "stop": 1751361198293, "uuid": "d5958e31-9d6a-444d-8367-a74db6c5c024", "historyId": "28f909ecd75e3205851eff6c17944d9b", "fullName": "User login functionality: Verify error message for login with incorrect credentials", "labels": [{"name": "severity", "value": "normal"}, {"name": "tag", "value": "invalid_user"}, {"name": "tag", "value": "login"}, {"name": "feature", "value": "User login functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}